mlrun/runtimes/function.py

```diff
@@ -378,9 +389,30 @@ def with_http(
         secret=None,
         worker_timeout: int = None,
         gateway_timeout: int = None,
+        trigger_name=None,
```

Comment:
This is a non backwards compatible change, why not adding it as the last kwarg ?

Reply:
talked offline, the `gateway_timeout` and the arguments coming after it recently added, so no BC worries